h2o-algos/src/main/java/hex/tree/SharedTreeModelWithContributions.java

```diff
@@ -3,6 +3,7 @@
 import hex.genmodel.algos.tree.SharedTreeNode;
 import hex.genmodel.algos.tree.SharedTreeSubgraph;
 import hex.genmodel.algos.tree.*;
+import hex.genmodel.attributes.parameters.KeyValue;
```

Comment:
it seems like this is still used, classes in `hex.genmodel.attributes` shouldn't be used in scoring logic

Reply:
I agree, thanks I am working on separating of scoring logic from genmodel class.

Reply:
Separated